### DISCUSSION TITLE
How can I remove the reference time in the transcript output file?

### DISCUSSION THREAD

**Discussion by palladinoandrea** [audacity/audacity#6436] [General]
I generated a text through transcription with openvino. Is there a way to remove the reference time from a transcribed sentence? for example in my output file? I find the line composed of: {initial time} {final time} sentence

**Comment by LWinterberg** [comment]
You can export it as SRT or VTT (instead of txt), in which case you'll be able to use the file in subtitle editors. Alternatively, you can import the TXT file as a Tab-Separated-Values-file (TSV) into a spreadsheet program such as Libreoffice, and then copy just the column with the text in it.

**Reply by palladinoandrea** [reply]
how you do it? it only lets me export in srt and vtt
